master/internal/core.go

```diff
@@ -859,13 +859,14 @@ func (m *Master) Run(ctx context.Context) error {
 
 	cert, err := m.config.Security.TLS.ReadCertificate()
 	if err != nil {
-		return errors.Wrap(err, "failed to read TLS certificate")
+		return err
```

Comment:
i could see this context being useful. what does it look like if you provide a non-existent TLS cert without it?

Reply:
forgot to push some changes, the new ones restore this error message and have some linting